docs/api.md

```diff
@@ -3727,6 +3728,9 @@ Contains the URL of the response.
 #### securityDetails.protocol()
 - returns: <[string]> String with the security protocol, eg. "TLS 1.2".
 
+#### securityDetails.sanList()
```

Comment:
Let's rename to `subjectAlternativeNames` to more closely match `subjectName`.

Reply:
Done